Frontend/ui-driver/src/Flow.purs

```diff
@@ -165,13 +165,17 @@ baseAppFlow baseFlow event = do
 
 checkVersion :: Int -> FlowBT String Unit
 checkVersion versioncode = do
-  when (getValueToLocalNativeStore IS_RIDE_ACTIVE /= "true" && versioncode < (getLatestAndroidVersion (getMerchant FunctionCall))) $ do
+  config <- getAppConfig Constants.appConfig
+  let appUpdateConfig = config.appUpdateConfig
+  when (getValueToLocalNativeStore IS_RIDE_ACTIVE /= "true" && versioncode < appUpdateConfig.appUpdateVersion) $ do
     lift $ lift $ doAff do liftEffect hideSplash
-    modifyScreenState $ AppUpdatePopUpScreenType (\appUpdatePopUpScreenState → appUpdatePopUpScreenState {updatePopup = AppVersion})
+    modifyScreenState $ AppUpdatePopUpScreenType (\appUpdatePopUpScreenState → appUpdatePopUpScreenState {updatePopup = AppVersion, appUpdateConfig = appUpdateConfig})
     fl <- UI.handleAppUpdatePopUp
     case fl of
       UpdateNow -> checkVersion versioncode
-      Later -> pure unit
+      Later -> do
+        void $ lift $ lift $ loaderText (getString LOADING) (getString PLEASE_WAIT_WHILE_IN_PROGRESS)
```

Comment:
@ashishsingh101 are we removing the toggle loader for all the scenarios ( for homescreenflow , onboardingflow ...)

Reply:
yes @MercyQueen